test-requirements.txt

```diff
@@ -7,7 +7,55 @@ pytest-forked
 pytest-mock
 pytest-xdist
 types-PyYAML
-
-# this gets installed as part of the collection requirements to avoid dependency conflicts
-# the readme installs this without dependencies to prevent conflicts
-# azure-cli==2.34.0
+packaging
```

Comment:
Not probably part of this PR, but wouldn't make sense to move test-requirements.txt into test/units/?

Reply:
This will be done in another pull request. Thanks